packages/esm-patient-orders-app/src/components/order-details-table.test.tsx

```diff
@@ -40,6 +43,33 @@ jest.mock('@openmrs/esm-patient-common-lib', () => {
   };
 });
 
+jest.mock('@openmrs/esm-framework', () => {
+  const originalModule = jest.requireActual('@openmrs/esm-framework');
+
+  return {
+    ...originalModule,
+    OpenmrsDatePicker: jest.fn(),
+  };
+});
```

Comment:
```suggestion
```

https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#you-should-not-need-to-use-partial-mocks